tensorflow/python/ops/logging_ops.py

```diff
@@ -72,13 +72,10 @@
                           "only a concern in graph mode. Below is an example "
                           "of how to ensure tf.print executes in graph mode:\n"
                           """```python
-    sess = tf.Session()
-    with sess.as_default():
-        tensor = tf.range(10)
-        print_op = tf.print(tensor)
-        with tf.control_dependencies([print_op]):
-          out = tf.add(tensor, tensor)
-        sess.run(out)
+    tensor = tf.range(10)
+    print_op = tf.print(tensor)
+    with tf.control_dependencies([print_op]):
```

Comment:
No control dependencies in tfv2

Reply:
this is document [control_dependencies](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/control_dependencies), also this code works with tfv2 nightly build.

Reply:
Yes but you don't need the control dependency if inside a tf.function or with eager execution enabled